Gemfile

```diff
@@ -2,17 +2,17 @@ ruby '2.3.3'
 
 source 'https://rubygems.org'
 
-gem 'avro-salsify-fork', '1.9.0.3', require: 'avro'
+gem 'avro-salsify-fork', '1.9.0.5', require: 'avro'
 gem 'grape'
 gem 'ice_nine', require: 'ice_nine/core_ext/object'
 gem 'pg'
 gem 'private_attr', require: 'private_attr/everywhere'
 gem 'puma'
-gem 'rails', '4.2.7'
-gem 'rails-api'
+gem 'rails', '5.0.1'
 
 group :test do
   gem 'json_spec'
+  gem 'rails-controller-testing'
```

Comment:
Yes, it's using `assert_template` in the specs for the static pages controller: https://github.com/salsify/avro-schema-registry/blob/master/spec/controllers/pages_controller_spec.rb